testsuite/smoke-test/src/client.rs

```diff
@@ -28,6 +33,107 @@ fn test_create_mint_transfer_block_metadata() {
     );
 }
 
+use std::time::Duration;
+use tokio::{runtime::Runtime, time::timeout};
+
+#[test]
+fn test_get_events_via_websocket_stream() {
+    let (_env, client) = setup_swarm_and_client_proxy(2, 1);
+
+    let currencies = client
+        .client
+        .get_currency_info()
+        .expect("Could not get currency info");
+
+    let rt = Runtime::new().unwrap();
+    let g = rt.enter();
+
+    let ms_500 = Duration::from_millis(500);
+
+    let ws_client = rt
+        .block_on(timeout(ms_500, client.websocket_client()))
+        .unwrap_or_else(|e| panic!("Timeout creating WebsocketClient: {}", e))
+        .unwrap_or_else(|e| panic!("Error connecting to WS endpoint: {}", e));
+
+    let (mut s_client, _) = rt
+        .block_on(timeout(ms_500, StreamingClient::new(ws_client, 10)))
+        .unwrap_or_else(|e| panic!("Timeout creating StreamingClient: {}", e));
```

Comment:
so far I only see tests on streaming client, I think only expose streaming client is better, creating 2 clients for use streaming client is unnecessary for user.